pkg/serviceprovider/github/githubclientbuilder.go

```diff
@@ -50,5 +50,19 @@ func (g *githubClientBuilder) createAuthenticatedGhClient(ctx context.Context, s
 	ctx = context.WithValue(ctx, oauth2.HTTPClient, g.httpClient)
 	ts := oauth2.StaticTokenSource(&oauth2.Token{AccessToken: tokenData.AccessToken})
 	lg.V(logs.DebugLevel).Info("Created new github client", "SPIAccessToken", spiToken)
+
+	// We need the githubBaseUrl in the githubClientBuilder struct to decide whether to create
```

Comment:
isn't it better to describe in an actual issue?

Reply:
Yea, and we already made a decision about it, so I will adjust the comment.